src/EmployeeRole.src.json

```diff
@@ -0,0 +1,59 @@
+{
+    "type": "EmployeeRole",
+    "uri": "http://schema4i.org/EmployeeRole",
+    "description": "A EmployeeRole describes employee relationship",
+    "links": [{
+        "url": "http://schema.org/EmployeeRole",
+        "description": "Original Schema.org type"
+    }],
+    "parents": [
+        { "@id": "http://schema4i.org/Person#HasOccupation" }
+    ],
+    "base": [
+        { "@id": "http://schema4i.org/Thing" }
```

Comment:
https://schema.org/EmployeeRole is based on https://schema.org/OrganizationRole, but if you don't need any attributes from there, it's no problem that you don't transfer `OrganizationRole` to S4i. It's also no problem to change this hierarchy in future by adding `OrganizationRole`. So this is ok for now.